agenta-backend/agenta_backend/celery_config.py

```diff
@@ -0,0 +1,23 @@
+import os
+from kombu import Exchange, Queue
+
+# Use environment variables with default values as fallback
+BROKER_URL = os.getenv('CELERY_BROKER_URL')
+CELERY_RESULT_BACKEND = os.getenv('CELERY_RESULT_BACKEND')
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_RESULT_SERIALIZER = 'json'
+CELERY_TIMEZONE = 'UTC'
+
+# TODO: Can we improve this to be more dynamic?
+CELERY_QUEUES = (
+    Queue('agenta_backend.tasks.evaluations.auto_exact_match',
```

Comment:
I don't see the advantage of having multiple tasks one per each eval. See comment in GoogleDoc. I think it would make us rerun the variant many times (+ no advantage)